charts/kube-master/templates/controller-manager.yaml

```diff
@@ -82,6 +83,12 @@ spec:
               readOnly: true
       containers:
         - name: controller-manager
+{{- if semverCompare ">=1.12-0" .Capabilities.KubeVersion.GitVersion }}
```

Comment:
can we change this to `">= 1.12"`? Thats what we are using in this deployment in a different location?